circle.yml

```diff
@@ -46,6 +46,19 @@ test:
     - mkdir -p ${CIRCLE_TEST_REPORTS}/{go,race}
     - "[ -f ${CIRCLE_ARTIFACTS}/test.log ] && docker run -i cockroachdb/cockroach-dev shell /go/bin/go2xunit < ${CIRCLE_ARTIFACTS}/test.log > ${CIRCLE_TEST_REPORTS}/go/test.xml"
     - "[ -f ${CIRCLE_ARTIFACTS}/testrace.log ] && docker run -i cockroachdb/cockroach-dev shell /go/bin/go2xunit < ${CIRCLE_ARTIFACTS}/testrace.log > ${CIRCLE_TEST_REPORTS}/race/testrace.xml"
+    - |
+      find "${CIRCLE_ARTIFACTS}" -name '*.log' -type f -exec \
+        grep -B 5 -A 10 -E '\-? *FAIL|^panic|^[Gg]oroutine \d+|(read|write) by.*goroutine|DATA RACE' {} ';' \
```

Comment:
The `\-? *` at the start of the regex looks redundant. 

It's sad that even after parsing the logs into a structured format with go2xunit, it's still easier to grep than to parse the xml.